apps/files_sharing/src/components/SharingInput.vue

```diff
@@ -169,19 +169,18 @@ export default {
 			}
 
 			const shareType = [
```

Comment:
Can't we do something like that?

```suggestion
			const shareType = Object.values(ShareType)
```